hashmap.py

```diff
@@ -0,0 +1,140 @@
+"""
+// Time Complexity : 
+    put - O(1)
+    get - O(1)
+    remove -  O(1)
+
+// Space Complexity : 
+    O(n) for using array
+// Did this code successfully run on Leetcode : N/A
```

Comment:
Try to run the problem on leetcode.
https://leetcode.com/problems/design-hashmap/

Reply:
Yeah, there was a bug which now I have fixed. However, I am not able to send a new pr.